packages/flutter_tools/lib/src/features.dart

```diff
@@ -102,8 +102,6 @@ const Feature flutterMacOSDesktopFeature = Feature(
   name: 'beta-quality support for desktop on macOS',
   configSetting: 'enable-macos-desktop',
   environmentOverride: 'FLUTTER_MACOS',
-  extraHelpText: flutterNext ?
-      'Newer beta versions are available on the beta channel.' : null,
```

Comment:
Make sure to leave the extra help text for this and the other desktop features

Reply:
Hmm, is that still a helpful thing to say though? What does "Newer beta versions are available on the beta channel" mean if you're on the dev channel?  I assume it references "beta-quality" but what does the beta channel have to do with it once it's available on all channels?

Reply:
That is the current correct messaging though - even though desktop is available on stable, it is meant to be a "beta preview"

Reply:
But if you're on the `dev` channel then what does `'Newer beta versions are available on the beta channel.` mean?
Anyway it's not really related to this change, I'll revert the `extraHelpText` part.

Reply:
Yeah, its certainly awkward. I dunno if we want to let the message be configured per-channel, but that would be one way to do it